libr/include/r_core.h

```diff
@@ -300,6 +300,7 @@ struct r_core_t {
 	char *lastcmd;
 	char *cmdlog;
 	bool cfglog; // cfg.corelog
+	bool use_newshell; // cfg.corelog
```

Comment:
we can remove this

Reply:
A new patch is on the top of the stack for this.